server/app/model.py

```diff
@@ -88,9 +89,9 @@ class Vector(Base):
     bsa1_overhang: str = Column(String, nullable=True)
     selection: str = Column(String, nullable=True)
     cloning_technique: str = Column(String, nullable=True)  # DNA or PCR synthesis?
-    is_BsmB1_free: str = Column(String, nullable=True)  # TODO: Might be removed...
```

Comment:
Put these back in please

Reply:
The TODO tags will be inserted again in the next commit.